NetworkingSampleApp/NetworkingSampleApp/Scenes/Users/User.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 
 /// Data structure of sample API user response
-struct User: Codable, Identifiable {
+struct User: Codable, Identifiable, Sendable {
```

Comment:
What do you mean by Sendable by default? 🤔

Reply:
@johnkodes 
`struct Test {}
extension Test: Sendable {}`

Test is sendable by default - there is no way to run in concurrency issue. Do we still need to mark it as Sendable?

Reply:
I suppose it is the same as with `Codable`. By default the struct is not `Codable` unless you say so, even though it might be. I suppose this is an optimization feature. 

Do I understand correctly to what you meant?

Reply:
@cejanen is right, the `Sendable` conformance is implicit for structs which include only parameters of `Sendable` types, hence we can remove the keyword here.

Reply:
Oh, nice, didn't know that. @cejanen good catch 🙇